inbm/dispatcher-agent/dispatcher/sota/os_updater.py

```diff
@@ -309,15 +311,15 @@ def update_remote_source(self, uri: Optional[CanonicalUri], repo: irepo.IRepo) -
         @param repo: Directory on disk where update has been downloaded, if given in manifest.
         @return: Command list to execute to perform update.
         """
-        pass
+        raise NotImplementedError()
```

Comment:
Could these raise, when we simply want it to just pass over and ignore the call?

Reply:
@nmgaston I know for sure these aren't implemented for Windows so we would want an error if we tried to call them.

Reply:
@nmgaston Oh, also with pass these will return None which the caller may not be able to handle.